test/fixtures/BootstrapCommand/version-mismatch/packages/package-2/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "package-2",
+  "version": "1.0.0",
+  "dependencies": {
+    "package-1": "^1.0.0",
```

Comment:
So how does this get mismatched in the first place? If you did a `lerna publish` choosing `prerelease`, that should have also bumped `package-2`'s version and its dependency on `package-1`. Also, it would have bumped the patch, since `1.0.0-alpha.x` will _always_ be "less than" `1.0.0`.